17.0 [MIG] fieldservice_sale

SMaciasOSI opened the pull request:
@max3903 @brian10048

osi-scampbell committed:
[ADD] fieldservice_sale

osi-scampbell committed:
[ADD] fieldservice_sale

osi-scampbell committed:
[ADD] fieldservice_sale

max3903 committed:
[REN] fsm_location.py

max3903 committed:
[FIX] View + icon

max3903 committed:
[FIX] View + icon

osimallen committed:
[ADD] Beta implementation of fieldservice_account

osimallen committed:
[FIX]PYLint errors

osimallen committed:
[ENH]Fixed issues from Max's review and implemented feedback changes

osimallen committed:
[FIX] PyLint Errors

osimallen committed:
[FIX]Minor fixes to views and data

Commit:
[FIX]add validation while adding contractor cost values without providing field service person

max3903 committed:
[IMP] fieldservice_account

max3903 committed:
[FIX] fieldservice_account

max3903 committed:
[FIX] No context field on ir.ui.view

max3903 committed:
[FIX] Cost computation

osi-scampbell committed:
[IMP] Bug Fixes

osi-scampbell committed:
[IMP] Bug Fixes

osi-scampbell committed:
[ADD] fieldservice_account: Vendor Bills Smart Button

murtuzasaleh committed:
[MIG][WIP][12.0] fieldservice_sale

murtuzasaleh committed:
[MIG][12.0] fieldservice_account

brian10048 committed:
[IMP] fieldservice_sale

Added ability to link a product with FSM order template and create FSM
orders upon sale order confirmation
Added option to install fieldservice_sale module

brian10048 committed:
[IMP] fieldservice_sale

Added support for selling recurring orders

brian10048 committed:
[FIX] fieldservice_sale errors

Travis
Typo, wrong model relationship
Improved values prepation method for recurring
Updated errors on SO smart button for recurring

osi-scampbell committed:
[IMP] Github Comments

max3903 committed:
[FIX] fieldservice_account: access rights

osi-scampbell committed:
[IMP] FSM Account Update

[IMP] Enable Dispatch/Managers to Enter Timesheets

[IMP] Overwrite Action to Remove Domain

Update ir_rule.xml

Update ir.model.access.csv

[IMP] Overwrite Record Rule

[IMP] Bill Customer on Location. Bill To Req/Show

osi-scampbell committed:
[IMP] FSM Sale Update

[IMP] Remove Duplicate Field

osi-scampbell committed:
[IMP] Autopopulate Description

osi-scampbell committed:
[IMP] Onoy Show Price to FSM Manager

max3903 committed:
[FIX] fieldservice_account: Bump version and provide migration script

kittiu committed:
[12.0][IMP] fieldservice_account, add test scripts

mourad-ehm committed:
[FIX] Bugs bill_to view and add default value to allow automatic creation of fsm.order from recuring for exemple

max3903 committed:
Update analytic_account.py

max3903 committed:
[FIX] fieldservice_account: migration script

hparfr committed:
Finish migration to v12 (forward port from v11)

osi-scampbell committed:
[IMP] Bug #266 False ETA Error

[IMP] Bug #265 Linked Location Search Error

[IMP] Bug #272 Rate Not Declared

[IMP] Bug #263 equipment_count Wrong

[IMP] Bug #277 Worker Pricelists Computed Wrong

[IMP] Flake8 + Remove Debug

osi-scampbell committed:
[IMP] Remove Unnecessary Loop.

max3903 committed:
[FIX] fieldservice_account: Wrong journals

max3903 committed:
[IMP] fieldservice_account

hparfr committed:
Add customer_id (moved from base module)

osi-scampbell committed:
[IMP] Add FSM Order to Vendor Bill List View

[IMP[ Make FSM Order Searchable

hparfr committed:
Rename __ini__.py to __init__.py

max3903 committed:
[FIX] fieldservice_account

max3903 committed:
[FIX] fieldservice_sale

max3903 committed:
[FIX] fieldservice_account

max3903 committed:
[IMP] fieldservice_vehicle_stock

max3903 committed:
[IMP] fieldservice_sale: Remove the recurring part

brian10048 committed:
[IMP] fieldservice_sale: Compute qty_delivered

- Completed field service orders will now update the qty_delivered field on linked SO line

brian10048 committed:
[IMP] fieldservice_sale: Add service policy

- This change will allow to create an FSM order for each sales order and link it to the delivery order and invoice

osi-scampbell committed:
[IMP] Fix Convert Error

brian10048 committed:
[IMP] fieldservice_sale: Compute qty_delivered

-  suggestions from reviewer comments

brian10048 committed:
[IMP] fieldservice_sale: Link invoices to FS order

clementmbr committed:
Autofill the FS location in Sale Orders

max3903 committed:
[IMP] fieldservice_sale: Merge and reuse fields

osi-scampbell committed:
[FIX] fieldservice_account: Extend FSM Account Wizard (#404)

brian10048 committed:
[IMP] fieldservice_sale

- Combine field service template info from sale order lines and add it to field service order

clementmbr committed:
Refactor autofill SO's FS location and add tests

brian10048 committed:
[IMP] fieldservice_account: Simplify module

- This change removes most of the invoicing logic from fieldservice_account and puts it into its own module

hparfr committed:
Clean up customer_id (billing), cost and change link between account_invoice* and fsm_order

hparfr committed:
add missing files

brian10048 committed:
[FIX] fieldservice_account: tests and travis

brian10048 committed:
[IMP] fieldservice_account: Update maintainers

OCA-git-bot committed:
fieldservice_account 12.0.2.0.2

brian10048 committed:
[FIX] fieldservice_sale

Consumable and Storable product types were not generating field services

max3903 committed:
[FIX] fieldservice_sale: Missing the template info

max3903 committed:
[FIX] fieldservice_sale: Missing location directions

max3903 committed:
[ADD] fieldservice_sale: Migration script to 12.0.2.0.0

brian10048 committed:
[FIX] fieldservice_sale_recurring

- Was not calling super properly and FSM orders on the sale line that were not recurring orders were not being created.
- Delete unused computed field.

murtuzasaleh committed:
[IMP][12.0] fieldservice_account

nikul-serpentcs committed:
[ADD] Added a fieldservice_route v12

max3903 committed:
[FIX] fieldservice_account: Show FSM smart button on vendor bills

OCA-git-bot committed:
fieldservice_account 12.0.2.0.3

max3903 committed:
[FIX] fieldservice_sale

max3903 committed:
[FIX] fieldservice_sale

max3903 committed:
[FIX] fieldservice_account

# Conflicts:
#	fieldservice_account/models/account_invoice.py

max3903 committed:
[IMP] fieldservice_sale

max3903 committed:
[FIX] fieldservice_sale

max3903 committed:
[FIX] fieldservice_sale: PEP8 errors

max3903 committed:
[IMP] fieldservice_route

[IMP] fieldservice_route

fieldservice_route 12.0.1.1.0

brian10048 committed:
[IMP] fieldservice_sale: Add Validation

brian10048 committed:
[FIX] fieldservice_sale: bug fixes

murtuzasaleh committed:
[FIX] Improved code.

max3903 committed:
[FIX] fieldservice_account

max3903 committed:
[FIX] fieldservice_account

OCA-git-bot committed:
fieldservice_account 12.0.3.0.1

brian10048 committed:
[FIX] fieldservice_sale

handle multiple sale order lines

max3903 committed:
[FIX] fieldservice_sale

max3903 committed:
[FIX] fieldservice_route

[FIX] fieldservice_route

osi-scampbell committed:
[IMP] Fix Travis Redlight

murtuzasaleh committed:
[FIX] Travis

max3903 committed:
[IMP] fieldservice_route

[FIX] fieldservice_route

max3903 committed:
[IMP] fieldservice_sale

# Conflicts:
#	fieldservice_sale/models/sale_order_line.py

Commit:
Update fieldservice_sale/models/fsm_order.py

Co-Authored-By: brian10048 <brian@mcmpest.com>

max3903 committed:
[IMP] fieldservice_account

OCA-git-bot committed:
fieldservice_account 12.0.3.0.2

max3903 committed:
[FIX] fieldservice_route

[FIX] fieldservice_route

fieldservice_route 12.0.1.2.0

osi-scampbell committed:
[IMP] FSO's on VB's

max3903 committed:
[ADD] Spanish (es) translation

brian10048 committed:
[FIX] fieldservice_sale

max3903 committed:
[FIX] fieldservice_account: Migration script

max3903 committed:
[FIX] fieldservice_account

max3903 committed:
[FIX] fieldservice_account

OCA-git-bot committed:
fieldservice_account 12.0.3.1.0

max3903 committed:
[FIX] fieldservice_route: Issue with locale

[FIX] fieldservice_route

Update test_fsm_order.py

brian10048 committed:
[FIX] fieldservice_sale

brian10048 committed:
[IMP] fieldservice_sale: Add test case

Tests multiple scenarios of selling products with field service tracking options

max3903 committed:
[ADD] fieldservice_route: Migration script

max3903 committed:
[IMP] fieldservice_route

max3903 committed:
[FIX] fieldservice_route

[FIX] fieldservice_route

[FIX] fieldservice_route: Comments

patrickrwilson committed:
[IMP] Add security groups

Add security groups for invoices smart button on FSO and FSO smart button on vendor bill. Also added security group to vendor bill list view.

OCA-git-bot committed:
fieldservice_account 12.0.3.1.1

brian10048 committed:
[FIX] fieldservice_sale

max3903 committed:
[IMP] fieldservice_route

fieldservice_route 12.0.2.1.0

[IMP] fieldservice_route

fieldservice_route 12.0.2.2.0

max3903 committed:
[FIX] fieldservice_sale: Access rights

max3903 committed:
[FIX] fieldservice_sale

max3903 committed:
[IMP] fieldservice_sale: Pricelist on branch

max3903 committed:
[FIX] fieldservice_sale: Singleton error

brian10048 committed:
[IMP] fieldservice_sale: black, isort, prettier

brian10048 committed:
[MIG] fieldservice_sale: Migration to 13.0

brian10048 committed:
[IMP] fieldservice_account: black, isort, prettier

brian10048 committed:
[MIG] fieldservice_account: Migration to 13.0

murtuzasaleh committed:
[IMP] Gave Access to portal user.

fieldservice_route 12.0.2.3.0

RLeeGML committed:
[FIX] fieldservice_sale migration changes

RLeeGML committed:
[FIX] invoice creation from SO

brian10048 committed:
[IMP] pre-commit, black

chienandalu committed:
[IMP] fieldservice: basic security

- Add a lower security group whose members only have access to their own
field service orders.
- Add related partner to location and worker forms to easily check the
relationship.

fieldservice_route 12.0.2.4.0

hparfr committed:
fieldservice_account fix relation between fsm_order and invoice_line

brian10048 committed:
[IMP] fieldservice_account: pre-commit
 by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: field-service-13.0/field-service-13.0-fieldservice_account
Translate-URL: https://translation.odoo-community.org/projects/field-service-13-0/field-service-13-0-fieldservice_account/

Freni-OSI committed:
Weblate (Turkish)

Currently translated at 100.0% (14 of 14 strings)

Translation: field-service-14.0/field-service-14.0-fieldservice_account
Translate-URL: https://translation.odoo-community.org/projects/field-service-14-0/field-service-14-0-fieldservice_account/tr/

Freni-OSI committed:
[FIX] Travis

Freni-OSI committed:
[14.0][MIG] fieldservice_sale

Freni-OSI committed:
[FIX] Travis

Freni-OSI committed:
[14.0][MIG] fieldservice_route

[UPD] Update fieldservice_route.pot

[UPD] README.rst

tate11 committed:
Added translation using Weblate (Turkish)

Translated using Weblate (Turkish)

Currently translated at 7.6% (5 of 65 strings)

Translation: field-service-14.0/field-service-14.0-fieldservice_route
Translate-URL: https://translation.odoo-community.org/projects/field-service-14-0/field-service-14-0-fieldservice_route/tr/

Translated using Weblate (Turkish)

Currently translated at 100.0% (65 of 65 strings)

Translation: field-service-14.0/field-service-14.0-fieldservice_route
Translate-URL: https://translation.odoo-community.org/projects/field-service-14-0/field-service-14-0-fieldservice_route/tr/

hparfr committed:
fieldservice_sale: remove required tpl in prd view

Create a blank fsm from a SO is a valid use case.
And it make more sense when used with fsm_recurring_sale

Implement rfc #735

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: field-service-14.0/field-service-14.0-fieldservice_account
Translate-URL: https://translation.odoo-community.org/projects/field-service-14-0/field-service-14-0-fieldservice_account/

pedrobaeza committed:
[FIX] fieldservice_account: Restrict views to authorized groups

Not putting groups on the view makes the rest of the users that access
to these views to receive a permission error.

It's also not enough to put the groups on the field, but the view.

pedrobaeza committed:
[FIX] fieldservice_sale: Restrict views to authorized users

Not putting groups on the view makes the rest of the users that access
to these views to receive a permission error.

It's also not enough to put the groups on the field, but the view.

mourad-ehm committed:
[IMP] add is_invoiceable on fsm stage

ibuioli committed:
Added translation using Weblate (Spanish (Argentina))

ibuioli committed:
Added translation using Weblate (Spanish (Argentina))

Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (65 of 65 strings)

Translation: field-service-14.0/field-service-14.0-fieldservice_route
Translate-URL: https://translation.odoo-community.org/projects/field-service-14-0/field-service-14-0-fieldservice_route/es_AR/

murtuzasaleh committed:
[14.0][FIX] fieldservice_sale

Vandan-OSI committed:
[15.0][FXI] Pre-commit

brian10048 committed:
[FIX] fieldservice_sale: adapt access to ir.actions.* records

victoralmau committed:
[MIG] fieldservice_route: Migration to 15.0

TT35715

Freni-OSI committed:
[15.0][MIG] fieldservice_account

Freni-OSI committed:
[15.0][MIG] fieldservice_sale

Commit:
[UPD] Update fieldservice_account.pot

OCA-git-bot committed:
[UPD] README.rst

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: field-service-15.0/field-service-15.0-fieldservice_account
Translate-URL: https://translation.odoo-community.org/projects/field-service-15-0/field-service-15-0-fieldservice_account/

Commit:
[UPD] Update fieldservice_route.pot

OCA-git-bot committed:
[UPD] README.rst

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: field-service-15.0/field-service-15.0-fieldservice_route
Translate-URL: https://translation.odoo-community.org/projects/field-service-15-0/field-service-15-0-fieldservice_route/

gbrito committed:
[MIG] fieldservice_account: Migration to 16.0

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

gbrito committed:
[MIG] fieldservice_route: Migration to 16.0

gbrito committed:
[16.0][IMP] fieldservice_account: Add tests

Commit:
[MIG] fieldservice_sale: Migration to 16.0
Co-Authored-By: Alexandre Fayolle, Stefan Ungureanu

Dependency on migration of fieldservice_account: https://github.com/OCA/field-service/pull/1042
Refactoring and fix view errors
Fix view error (obsolete XMLID)
Remove obsolete onchange
Update unit tests

Commit:
[UPD] Update fieldservice_account.pot

OCA-git-bot committed:
d at 100.0% (10 of 10 strings)

Translation: field-service-16.0/field-service-16.0-fieldservice_account
Translate-URL: https://translation.odoo-community.org/projects/field-service-16-0/field-service-16-0-fieldservice_account/es/

brian10048 committed:
[IMP] fieldservice_sale: Simplify FSM creation

Sale order and sale order line shared similar code.  This simplifies the logic needed to create FSM orders from a sale order while maintaining the same results

brian10048 committed:
[IMP] fieldservice_sale: Clean up tests

Additions made to tests in migration to 15 (5a3201c) do not make sense.  They just call certain methods, but assert nothing after the method call

Translated using Weblate (Spanish)

Currently translated at 100.0% (29 of 29 strings)

Translation: field-service-15.0/field-service-15.0-fieldservice_sale
Translate-URL: https://translation.odoo-community.org/projects/field-service-15-0/field-service-15-0-fieldservice_sale/es/

Translated using Weblate (Spanish (Chile))

Currently translated at 100.0% (29 of 29 strings)

Translation: field-service-15.0/field-service-15.0-fieldservice_sale
Translate-URL: https://translation.odoo-community.org/projects/field-service-15-0/field-service-15-0-fieldservice_sale/es_CL/

Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (29 of 29 strings)

Translation: field-service-15.0/field-service-15.0-fieldservice_sale
Translate-URL: https://translation.odoo-community.org/projects/field-service-15-0/field-service-15-0-fieldservice_sale/es_AR/

Translated using Weblate (Italian)

Currently translated at 100.0% (29 of 29 strings)

Translation: field-service-15.0/field-service-15.0-fieldservice_sale
Translate-URL: https://translation.odoo-community.org/projects/field-service-15-0/field-service-15-0-fieldservice_sale/it/

Translated using Weblate (Italian)

Currently translated at 100.0% (29 of 29 strings)

Translation: field-service-15.0/field-service-15.0-fieldservice_sale
Translate-URL: https://translation.odoo-community.org/projects/field-service-15-0/field-service-15-0-fieldservice_sale/it/

paradoxxxzero committed:
[IMP] fieldservice_sale: Allow overrides in _field_service_generation

[BOT] post-merge updates

dalonsod committed:
[FIX] fieldservice_sale: duplicated sales order doesn't create FSM Order

As sales order line link to order is preserved when duplicating a Sales Order,
the FSM Order generation when confirming the sales order is wrongly skipped.
This commit fix it.

Translated using Weblate (Italian)

Currently translated at 100.0% (29 of 29 strings)

Translation: field-service-15.0/field-service-15.0-fieldservice_sale
Translate-URL: https://translation.odoo-community.org/projects/field-service-15-0/field-service-15-0-fieldservice_sale/it/

yankinmax committed:
[FIX] fieldservice_sale: fsm_location_id is mandatory
only if order line display_type not in ('line_section', 'line_note')

Commit:
[UPD] Update fieldservice_sale.pot

OCA-git-bot committed:
[UPD] README.rst

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: field-service-16.0/field-service-16.0-fieldservice_sale
Translate-URL: https://translation.odoo-community.org/projects/field-service-16-0/field-service-16-0-fieldservice_sale/

Translated using Weblate (Italian)

Currently translated at 100.0% (29 of 29 strings)

Translation: field-service-16.0/field-service-16.0-fieldservice_sale
Translate-URL: https://translation.odoo-community.org/projects/field-service-16-0/field-service-16-0-fieldservice_sale/it/

Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (29 of 29 strings)

Translation: field-service-16.0/field-service-16.0-fieldservice_sale
Translate-URL: https://translation.odoo-community.org/projects/field-service-16-0/field-service-16-0-fieldservice_sale/es_AR/

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (29 of 29 strings)

Translation: field-service-16.0/field-service-16.0-fieldservice_sale
Translate-URL: https://translation.odoo-community.org/projects/field-service-16-0/field-service-16-0-fieldservice_sale/pt_BR/

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (29 of 29 strings)

Translation: field-service-16.0/field-service-16.0-fieldservice_sale
Translate-URL: https://translation.odoo-community.org/projects/field-service-16-0/field-service-16-0-fieldservice_sale/pt_BR/

Translated using Weblate (Spanish)

Currently translated at 100.0% (29 of 29 strings)

Translation: field-service-16.0/field-service-16.0-fieldservice_sale
Translate-URL: https://translation.odoo-community.org/projects/field-service-16-0/field-service-16-0-fieldservice_sale/es/

Commit:
[UPD] Update fieldservice_route.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

Translated using Weblate (Italian)

Currently translated at 100.0% (29 of 29 strings)

Translation: field-service-16.0/field-service-16.0-fieldservice_sale
Translate-URL: https://translation.odoo-community.org/projects/field-service-16-0/field-service-16-0-fieldservice_sale/it/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: field-service-16.0/field-service-16.0-fieldservice_route
Translate-URL: https://translation.odoo-community.org/projects/field-service-16-0/field-service-16-0-fieldservice_route/

DiegoFJ95 committed:
[IMP] fieldservice_route: pre-commit auto fixes

DiegoFJ95 committed:
[MIG] fieldservice_route: Migration to 17.0

Commit:
[FIX] pre-commit

brian10048 committed:
[IMP] fieldservice_account

Update variable name for clarity

Commit:
[UPD] Update fieldservice_account.pot

[BOT] post-merge updates

weblate committed:
at 100.0% (12 of 12 strings)

Translation: field-service-16.0/field-service-16.0-fieldservice_account
Translate-URL: https://translation.odoo-community.org/projects/field-service-16-0/field-service-16-0-fieldservice_account/es/

SMaciasOSI committed:
[IMP] fieldservice_sale: pre-commit auto fixes

SMaciasOSI committed:
[MIG] fieldservice_sale: Migration to 17.0

SMaciasOSI committed:
[IMP] fieldservice_account: pre-commit auto fixes

SMaciasOSI committed:
[MIG] fieldservice_account: Migration to 17.0

SMaciasOSI committed:
[ADD] fsm_order.py: def message_post_with_view

SMaciasOSI committed:
[FIX] pre-commit

brian10048 commented:
/ocabot migration fieldservice_sale

SMaciasOSI committed:
[FIX] pre-commit

SMaciasOSI committed:
[FIX] replace message_post_with_view with message_mail_with_source

SMaciasOSI commented:
This PR requires #1150 to be merged first so the test can pass after a rebase

brian10048 committed:
[FIX] fieldservice: FSM Person Calendar Filter

OCA-git-bot committed:
Merge PR #1142 into 17.0

Signed-off-by dreispt

Commit:
[UPD] Update fieldservice.pot

Commit:
[UPD] Update fieldservice_route.pot

dreispt commented on fieldservice_sale/models/sale_order.py:
```diff
@@ -149,15 +149,15 @@ def _field_service_generate(self):
-            lambda l: l.product_id.field_service_tracking == "sale"
-            and not l.fsm_order_id
+            lambda L: L.product_id.field_service_tracking == "sale"
+            and not L.fsm_order_id
```
Why the uppercase? Convention is to use lowercase variable names.

SMaciasOSI replied:
> Why the uppercase? Convention is to use lowercase variable names.

With the lowecase I got an error regarding "ambiguous variable name". I got the suggestion to change from l to L from here: https://docs.astral.sh/ruff/rules/ambiguous-variable-name/

OCA-git-bot committed:
[BOT] post-merge updates

SMaciasOSI committed:
[IMP] v17 style icon

SMaciasOSI committed:
[IMP] v17 style icon

SMaciasOSI committed:
[ADD] v17 style icon in svg

SMaciasOSI committed:
[ADD] from v14: _get_invoiceable_stage method & fsm_order_ids field to invoice line

SMaciasOSI committed:
Merge branch '17.0-imp-v17-style-icon' of github.com:ursais/field-service into 17.0-imp-v17-style-icon

OCA-git-bot committed:
Merge PR #1154 into 17.0

Signed-off-by max3903

OCA-git-bot committed:
Merge PR #1150 into 17.0

Signed-off-by brian10048

Commit:
[UPD] Update fieldservice.pot

Commit:
[UPD] Update fieldservice_account.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1153 into 17.0

Signed-off-by brian10048

Commit:
[UPD] Update fieldservice.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: field-service-17.0/field-service-17.0-fieldservice_route
Translate-URL: https://translation.odoo-community.org/projects/field-service-17-0/field-service-17-0-fieldservice_route/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: field-service-17.0/field-service-17.0-base_territory
Translate-URL: https://translation.odoo-community.org/projects/field-service-17-0/field-service-17-0-base_territory/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: field-service-17.0/field-service-17.0-fieldservice
Translate-URL: https://translation.odoo-community.org/projects/field-service-17-0/field-service-17-0-fieldservice/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (12 of 12 strings)

Translation: field-service-17.0/field-service-17.0-fieldservice_account
Translate-URL: https://translation.odoo-community.org/projects/field-service-17-0/field-service-17-0-fieldservice_account/it/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (564 of 564 strings)

Translation: field-service-17.0/field-service-17.0-fieldservice
Translate-URL: https://translation.odoo-community.org/projects/field-service-17-0/field-service-17-0-fieldservice/es/

SMaciasOSI committed:
Merge branch '17.0-mig-fieldservice_sale' of github.com:ursais/field-service into 17.0-mig-fieldservice_sale

santiagordz committed:
[FIX] field_service_sale

santiagordz committed:
[FIX] fieldservice_sale
